### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by TimoKramer**
Please add missing parts to the release notes.
@grischoun can you add some notes on the changed behaviour of the upsert?

### CODE REVIEW

**Comment by grischoun** [comment]
@TimoKramer You are talking about adding something in the 'release_notes.md', right?
I see this '#62 Add upsert to improve transact performance' already there.
I would slightly change the title to 'Change upsert implementation to improve transaction performance'. But other than that what would you like to see in there?
From an external user point of view there is no change in behaviour.
I could say that the implementation is now done entirely at a lower level, i.e. at the hitchhiker level and no longer at a higher level in the Datahike stack but it seems to be an implementation detail and I don't think it should go in a release note.

**Comment by TimoKramer** [comment]
> I would slightly change the title to 'Change upsert implementation to improve transaction performance'. 
Please feel free to change it.

> But other than that what would you like to see in there?
Just a short note on the changes to the tx-report, maybe some mentioning on the performance improvements made.

**Comment by grischoun** [comment]
@TimoKramer I have added comments to #201 

> > I would slightly change the title to 'Change upsert implementation to improve transaction performance'.
> 
> Please feel free to change it.
> 
> > But other than that what would you like to see in there?
> 
> Just a short note on the changes to the tx-report, maybe some mentioning on the performance improvements made.

@TimoKramer  It is done.

**Review by grischoun — APPROVED** [review]
Sounds good.

**Comment by TimoKramer** [comment]
Added my notes on api namespace as well

**Comment by TimoKramer** [comment]
@kordano rebased on development and squashed. Set up to be merged into development. Could be merged into master as well, if you want.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+
+With this release, you would only see the newly added entry and no information about retraction or addition is shown (it is assumed to be an addition).
+```
+#datahike.db.TxReport{
+...
+:tx-data [#datahike/Datom [1 :name "Petr" 536870914]]
+...}
+```
+
+Thanks to all the contributors and the community for helping on this release, namely Judith, Pablo, Wade, Chrislain, Timo, Konrad and Christian. Special thanks go to [clojurists together](https://www.clojuriststogether.org/) for funding large parts of this work.
```
Comment (kordano) [inline CHANGELOG.md:34]: It is clear that we together are working on the project. I would add external people that contributed through PRs.

**Review by kordano — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (TimoKramer) [reply CHANGELOG.md:34]: so me then:) ... just kidding. I can remove it.

**Review by kordano — APPROVED** [review]

**Pull request merged**